### PR TITLE
fix owner ut

### LINKED ISSUE #121: Some unit tests for module Owner are not running
### 1. Describe the bug

Because of losing  func Test(t *testing.T), some unit tests for module Owner are not running.

### 2. Minimal reproduce step (Required)

> cd cdc/cdc/owner
> go test

### 3. What did you see instead (Required)
Some unit tests don't run;

### 4. What did you expect to see? (Required)
all unit tests run;

### 5. What is your migration tool and TiKV version? (Required)

- TiKV CDC:

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What problem does this PR solve?

Issue Number: close #121 

Problem Description: 

Some unit tests in model Owner don't run.

### What is changed and how does it work?

Add  func Test(t *testing.T)

### Code changes

- Has exported function/method change

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test

### CODE REVIEW

File: cdc/cdc/owner/scheduler_v1_test.go [hunk 1]
```diff
@@ -123,9 +132,18 @@ func (s *schedulerSuite) TestScheduleOneCapture(c *check.C) {
 	c.Assert(err, check.IsNil)
 	c.Assert(shouldUpdateState, check.IsFalse)
 	s.tester.MustApplyPatches()
+
+	for spanid, info := range s.state.TaskStatuses[captureID].KeySpans {
```
Comment (pingyu) [inline cdc/cdc/owner/scheduler_v1_test.go:136]: For debug ? Suggest to remove debug codes.

**Review by pingyu — DISMISSED** [review]
Rest LGTM~

**Review by haojinming — DISMISSED** [review]
LGTM

**Review by pingyu — APPROVED** [review]
LGTM~

**Pull request merged**